RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHBHE_cfi.py

```diff
@@ -75,3 +75,19 @@
 run3_HB.toModify(particleFlowRecHitHBHEOnly,
     producers = { 0: dict(src = "hbhereco") }
 )
+
+# Alpaka and Legacy
+particleFlowRecHitHBHEOnlyLegacy = particleFlowRecHitHBHEOnly.clone()
+from RecoParticleFlow.PFRecHitProducer.legacyPFRecHitProducer_cfi import legacyPFRecHitProducer as _legacyPFRecHitProducer
+
+legacyPFRecHitProducer = _legacyPFRecHitProducer.clone(
+        src = "pfRecHitSoAProducerHCAL"
+    )
+
+legacyPFRecHitProducerHBHEOnly = _legacyPFRecHitProducer.clone(
```

Comment:
Also here either prepend this module and `legacyPFRecHitProducer` with underscore, or pass the result of `clone()` directly to `alpaka.toReplaceWith()`.